.circleci/config.yml

```diff
@@ -46,6 +53,17 @@ jobs:
           paths:
             - ./node_modules
 
+  solidity-solcjs-ext-test:
+    docker:
+      - image: circleci/node
+    steps:
+      - show-npm-version
+      - checkout:
+          path: solc-js/
+      - run: git clone --depth 1 "https://github.com/ethereum/solidity" solidity/
+      - run: cd solidity/ && curl "https://binaries.soliditylang.org/bin/soljson-nightly.js" --location --output soljson.js
+      - run: cd solidity/ && test/externalTests/solc-js/solc-js.sh "$(realpath soljson.js)" "$(scripts/get_version.sh)" "$(realpath ../solc-js/)"
```

Comment:
This is the part that depends on https://github.com/ethereum/solidity/pull/12594. It will not fail without it but the third argument will be ignored and the test will actually clone `master` instead of using the local checkout of solc-js we give it.